### PR TITLE
10 mar

### PR DESCRIPTION
**Pull request by MarianBastiurea**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,6 +25,12 @@
       <version>5.9.2</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
```
Comment (andreiursudev) [inline pom.xml]: remove this dependency

File: src/main/java/org/example/c11/LambdaExpression/List.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    public static void main(String[] args) {
+
+        String[] names = {"Anna", "Bob", "Carole", "David", "Ed", "Fred", "Gary"};
+
+        Arrays.setAll(names, i -> names[i].toUpperCase());
+        System.out.println("--> Transform to Uppercase");
+        System.out.println(Arrays.toString(names));
+
+        java.util.List<String> backedByArray = Arrays.asList(names);
```
Comment (andreiursudev) [inline src/main/java/org/example/c11/LambdaExpression/List.java]: use `import java.util.List;`

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/c11/LambdaExpression/List.java (hunk 2 above)
Comment (MarianBastiurea) [reply src/main/java/org/example/c11/LambdaExpression/List.java]: done

Comment (andreiursudev) [reply src/main/java/org/example/c11/LambdaExpression/List.java]: Replace ` java.util.List` with `List`

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**